package.json

```diff
@@ -51,7 +51,7 @@
     "postcss": "^7.0.23",
     "postcss-modules-extract-imports": "^2.0.0",
     "postcss-modules-local-by-default": "^3.0.2",
-    "postcss-modules-scope": "^2.1.1",
+    "postcss-modules-scope": "css-modules/postcss-modules-scope#master",
```

Comment:
I think we already do release, or I am wrong?

Reply:
https://github.com/css-modules/postcss-modules-scope/releases

![image](https://user-images.githubusercontent.com/11512544/76916483-7d826900-68fb-11ea-8403-36d243fcc984.png)

I saw the latest version is `2.1.1` released on 28 Nov 2019. But this option is merged into master on 26 Feb 2020.
I also checked the released source code. There is no `exportGlobals` option there.

Reply:
Hi @evilebottnawi ,
Could you help me to release a new version?
After releasing, I will change `postcss-modules-scope` version in package.json.

Thanks.

Reply:
Yes, in my TODO, let's do it tomorrow

Reply:
Done https://github.com/css-modules/postcss-modules-scope/releases/tag/v2.2.0

Reply:
Ok thx